official/docs/csharp/current/pickups/list.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace EasyPostExamples
+{
+    public class Examples
+    {
+        public static async Task Main()
+        {
+            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY");
+
+            EasyPost.Client client = new EasyPost.Client(apiKey);
+
+            EasyPost.Models.API.PickupCollection pickups = await client.Pickup.All();
```

Comment:
```suggestion
            PickupCollection pickups = await client.Pickup.All();
```

We never explicit the namespaces in the API doc, we could remove this. We should also add `page_size` to the param to match other `all` function code examples.

Reply:
Sounds good, was trying to keep it as close to what the code generator made out of the box, but clearly needs some modification.